assets/page/SettingsAccount.svelte

```diff
@@ -83,6 +84,10 @@ function updateUserFromForm(e) {
       <span slot="label">{l('Ignore join/part messages')}</span>
     </Checkbox>
 
+    <Checkbox name="compact" bind:checked="{compactDisplay}">
+      <span slot="label">{l('Enable compact message display')}</span>
+    </Checkbox>
```

Comment:
I think this checkbox should be moved under "Color scheme".

Reply:
Done.